config/blazer.yml

```diff
@@ -0,0 +1,3 @@
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
```

Comment:
Something in `.env.example` for this?

Reply:
Fixed it so that it'll work in development out of the box.